lib/allora/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
```

Comment:
When I tried deploying I kept getting errors saying it couldn't find `process.env`. I resolved that by adding `   "types": ["node"],` to the compilerOptions. Curious if you managed to deploy it without adding that field?

Reply:
the fact it worked for me might have been environment pollution on my part. Good call though, i'll add that.